Gemfile.lock

```diff
@@ -1,190 +1,209 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actioncable (6.1.6.1)
-      actionpack (= 6.1.6.1)
-      activesupport (= 6.1.6.1)
+    actioncable (6.1.7.8)
```

Comment:
There is no need to modify `Gemfile.lock`. You can remove it from the PR. Next time, please ensure this file is not in the committed files.